addons/html_editor/static/src/main/media/image_plugin.js

```diff
@@ -227,6 +227,18 @@ export class ImagePlugin extends Plugin {
                 if (!selectedImg) {
                     return;
                 }
+                const toggleClasses = (elementName, classToRemove) => {
+                    const element = document.querySelector(`[name="${elementName}"]`);
+                    if (element && element.classList.contains('active')) {
+                        element.classList.remove('active');
+                        selectedImg.classList.remove(classToRemove);
+                    }
+                };
+                if (command === 'SHAPE_ROUNDED') {
+                    toggleClasses('shape_circle', commandToClassNameDict['SHAPE_CIRCLE']);
+                } else if (command === 'SHAPE_CIRCLE') {
+                    toggleClasses('shape_rounded', commandToClassNameDict['SHAPE_ROUNDED']);
+                }
```

Comment:
Rather than this I would suggest to move `SHAPE_ROUNDED` and `SHAPE_CIRCLE` to its own case and keep a `mutuallyExclusiveShapes` object.
```js
const mutuallyExclusiveShapes = {
    SHAPE_ROUNDED: "rounded-circle",
    SHAPE_CIRCLE: "rounded",
};
```
and from `selectedImg` you remove `mutuallyExclusiveShapes[command]` class and toggle `commandToClassNameDict[command]`.

Also test cases are missing.